lib/providers/restful-provider.js

```diff
@@ -0,0 +1 @@
+module.exports = function(){};
```

Comment:
no, that will crash. the backoff logic creates an instance of a restful provider if the ws provider fails x times. so for now, this is just nothing